.claude/skills/issue.md

```
@@ -0,0 +1,178 @@
+
+## 브랜치 네이밍 컨벤션
+
+```
+{type}/{scope}/{이슈번호}-{kebab-case-영어설명}
+```
+
+- `scope`: `recruit` / `homepage` / `root` / `ui` — 이슈 내용에 맞는 스코프 선택
+- `설명`: 이슈 제목을 **영어**로 의미 번역 후 kebab-case 변환 (한국어 그대로 금지)
+- 최대 50자 이내로 축약
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)